source/sync/details/conflict-resolution.txt

```diff
@@ -4,17 +4,19 @@
 Conflict Resolution
 ===================
 
-.. default-domain:: mongodb
+.. meta::
+   :description: A high-level overview of how Atlas Device Sync resolves synced data conflicts.
```

Comment:
I might instead position this benefit-first, as conflict resolution is one of the reasons to use Realm/Device Sync. Maybe something like:

```suggestion
   :description: Learn about how Atlas Device Sync automatically resolves synced data conflicts, or how to build custom conflict resolution rules.
```